cefpanda/__init__.py

```diff
@@ -108,11 +108,26 @@ def __init__(self):
         self._cef_texture.set_format(p3d.Texture.FRgba4)
 
         card_maker = p3d.CardMaker("browser2d")
-        card_maker.set_frame(-self._UI_SCALE, self._UI_SCALE, -self._UI_SCALE, self._UI_SCALE)
+        if size is None:
```

Comment:
If `size` is `None`, we can fill it up like we do for `ui_scale` and then share the `set_frame()` and `self._size` logic.

Reply:
Done. Retained the name _size for now, since it doesn't relate to a scaling operation.